warehouse/query-core/src/main/java/datawave/query/jexl/visitors/LiteralNodeSubsetVisitor.java

```diff
@@ -42,4 +45,23 @@ public Object visit(ASTEQNode node, Object data) {
         return literals;
     }
     
+    // Descend through these nodes
+    @Override
+    public Object visit(ASTJexlScript node, Object data) {
+        node.childrenAccept(this, data);
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTERNode node, Object data) {
```

Comment:
Do we need to implement this for the ASTNRNode too? (Is that the "not equals regex" node?)

Reply:
Oversight on my part.